src/Symfony/Bundle/TwigBundle/DependencyInjection/Compiler/ExtensionPass.php

```diff
@@ -63,6 +64,10 @@ public function process(ContainerBuilder $container)
             $container->getDefinition('twig.extension.code')->replaceArgument(0, $container->getParameter('templating.helper.code.file_link_format'));
         }
 
+        if (!$container->has('security.token_storage')) {
```

Comment:
Ah, now I get what you are trying to achieve. But then, the service definition requested via `getDefinition()` below should be `twig.app_variable` and not `security.token_storage`.